content/md/en/docs/maintain/monitor.md

```diff
@@ -4,59 +4,174 @@ description:
+
+To see the list of columns and modify which columns are displayed, click **Settings**.
+
+![Click Settings to modify the visible columns](/media/images/docs/list-view-control.png)
+
+The columns you can toggle on and off include:
+
+- **Validator** to display an indicator to differentiate validator nodes from other nodes. 
+- **Location** to display the location of a node.
+- **Implementation** to display the version of the software running on the node.
+- **Network ID** to display the public network identifier for the node.
+- **Peer Count** to display the number of peers connected to the node.
+- **Transactions in Queue** to display the number of transactions in the Ready queue waiting for a block author.
+- **Upload Bandwidth** to display a graph of recent upload activity in MB per second.
+- **Download Bandwidth** to display a graph of recent download activity in MB per second.
+- **State Cache Size** to display a graph of the state cache size in MB.
+- **Block** to display the current best block number for the node to ensure it's staying synchronized with its peers.
+- **Block Hash** to display the block hash for the current best block number for the node.
+- **Finalized Block** to display the most recently finalized block number for the node to ensure it's staying synchronized with its peers.
+- **Finalized Block Hash** to display the block hash for the most recently finalized block number for the node.
+- **Block Time** to display the block execution time between blocks.
+- **Block Propagation Time** to display the time it took for the node to import the most recent block.
+- **Last Block Time** to display the time it took for the node to author the most recent block.
+- **Node Uptime** to display the number of days the node has been online without being restarted.
+
+### Pin node information
+
+You can keep the information for one or more nodes in view in the dashboard by clicking the line displayed for the node in the dashboard list.
+For example, if you want to keep the information for two nodes visible while the information for other nodes scrolls by, you can select the nodes in the list and pin the information in place.
+
+![Keep information for selected nodes displayed](/media/images/docs/pin-nodes.png)
+
+### Display nodes on a map
+
+As an alternative to the list of nodes, you can display nodes on a global map.
+The map provides a visual representation of where nodes are located, where blocks are being produced and imported in real time, and where there are higher concentrations of nodes geographically.
+
+To see represented on a map, click **Map**.
+
+![Display nodes on a global map](/media/images/docs/node-map-view.png)
+
+In this view, you can hover over any node location on the map to display a subset of the information available for the node, including the node name, location, best block number, best block hash, and block time statistics.
```

Comment:
I wonder whether it's worth noting that nodes taht do not have a known location will not be displayed in this view.